components/builder-api/src/server/resources/origins.rs

```diff
@@ -504,7 +505,7 @@ fn list_origin_secrets(req: HttpRequest<AppState>) -> HttpResponse {
         Ok(list) => {
             // Need to map to different struct for hab cli backward compat
             let new_list: Vec<OriginSecretWithOriginId> =
-                list.into_iter().map(|s| s.into()).collect();
+                list.into_iter().map(Into::into).collect();
```

Comment:
I like using the type rather than the trait because I think it makes the code clearer
```suggestion
                list.into_iter().map(OriginSecret::into).collect();
```

Reply:
Another alternative:
```rust
            let new_list: Vec<_> = list.into_iter()
                                       .map(OriginSecretWithOriginId::from)
                                       .collect();
```